enterprise/ha/src/main/java/org/neo4j/kernel/ha/transaction/OnDiskLastTxIdGetter.java

```diff
@@ -33,11 +33,15 @@ public OnDiskLastTxIdGetter( Supplier<NeoStores> neoStoresSupplier )
         this.neoStoresSupplier = neoStoresSupplier;
     }
 
+    // This method is used to construct credentials for election process.
+    // And can be invoked at any moment of instance lifecycle.
+    // It mean that its possible that we will be invoked when neo stores are stopped
+    // (for example while we copy store)
     @Override
     public long getLastTxId()
     {
         TransactionIdStore neoStore = getNeoStores().getMetaDataStore();
-        return neoStore.getLastCommittedTransactionId();
+        return neoStore != null ? neoStore.getLastCommittedTransactionId() : TransactionIdStore.BASE_TX_ID;
```

Comment:
Are you sure it's the meta store that is null? That looks odd. I'd expect it to be the `NeoStores`, i.e. the `neoStoresSupplier.get()` that would return null. Have you verified that?